.prettierignore

```diff
@@ -0,0 +1,2 @@
+dist
+pnpm-lock.yaml
```

Comment:
So that's why there was an issue with the lock file! For some reason it changes after installing any tiny package. Thank you for fixing this.